packages/SwingSet/test/gc/vat-target.js

```diff
@@ -10,5 +13,13 @@ export function buildRootObject() {
     makeInvitationTarget(zoe) {
       return E(zoe).makeInvitationZoe();
     },
+
+    store: x => {
+      baggage.init('x', x);
+      assert(ws);
+      ws.add(x);
+    },
+    dummy: () => 0,
+    drop: () => (ws = undefined),
```

Comment:
I'll use `storeInWS`, to avoid confusion with the `storeInWeakSet` provided by the neighboring bootstrap vat (and used in a `queueToVatRoot` in the actual `gc-vat.test.js`). The latter calls the former, but must be called from the bootstrap vat rather than a `queueToVatRoot`.

Reply:
Actually, I think I'll just add comments to that portion of vat-target.js to point out that all three methods are for the benefit of 9939. The fact that the `store()` method necessarily stores it in *both* `baggage` *and* the WeakSet means `storeInWeakSet` or `storeInWS` would be inaccurate.